qa/integration-tests/src/test/java/io/camunda/it/migration/util/CamundaMigrator.java

```diff
@@ -0,0 +1,302 @@
+        put("CAMUNDA_TASKLIST_ZEEBE_RESTADDRESS", "http://camunda:8080");
+        put("CAMUNDA_TASKLIST_ELASTICSEARCH_INDEXPREFIX", indexPrefix + "-tasklist");
+        put("CAMUNDA_TASKLIST_ZEEBEELASTICSEARCH_PREFIX", indexPrefix);
+        put("CAMUNDA_OPERATE_ELASTICSEARCH_URL", "http://elasticsearch:9200");
+        put("CAMUNDA_OPERATE_ELASTICSEARCH_INDEXPREFIX", indexPrefix + "-operate");
+        put("CAMUNDA_OPERATE_ZEEBEELASTICSEARCH_PREFIX", indexPrefix);
+        put("CAMUNDA_OPERATE_ZEEBEELASTICSEARCH_URL", "http://elasticsearch:9200");
+        put("CAMUNDA_OPERATE_ZEEBE_GATEWAYADDRESS", "camunda:26500");
+        put("ZEEBE_BROKER_GATEWAY_ENABLE", "true");
+        put("CAMUNDA_DATABASE_URL", "http://elasticsearch:9200");
+      }
+    };
+  }
+
+  private Map<String, String> opensearchConfiguration87() {
+    return new HashMap<>() {
+      {
+        put("SPRING_PROFILES_ACTIVE", "tasklist,broker,auth,operate");
+        put("ZEEBE_BROKER_CLUSTER_PARTITIONSCOUNT", "1");
+        put("ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME", OpensearchExporter.class.getName());
+        put("ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL", "http://opensearch:9200");
+        put("ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_BULKSIZE", "1");
+        put("ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_INDEX_PREFIX", indexPrefix);
+        put("CAMUNDA_TASKLIST_ZEEBE_COMPATIBILITYMODE", "true");
+        put("CAMUNDA_TASKLIST_DATABASE", "opensearch");
+        put("CAMUNDA_TASKLIST_OPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_TASKLIST_OPENSEARCH_INDEXPREFIX", indexPrefix + "-tasklist");
+        put("CAMUNDA_TASKLIST_ZEEBEOPENSEARCH_PREFIX", indexPrefix);
+        put("CAMUNDA_TASKLIST_ZEEBEOPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_TASKLIST_ZEEBE_GATEWAYADDRESS", "camunda:26500");
+        put("CAMUNDA_TASKLIST_ZEEBE_RESTADDRESS", "http://camunda:8080");
+        put("CAMUNDA_OPERATE_OPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_OPERATE_DATABASE", "opensearch");
+        put("CAMUNDA_OPERATE_ZEEBEOPENSEARCH_URL", "http://opensearch:9200");
+        put("CAMUNDA_OPERATE_ZEEBE_GATEWAYADDRESS", "camunda:26500");
+        put("CAMUNDA_OPERATE_OPENSEARCH_INDEXPREFIX", indexPrefix + "-operate");
+        put("CAMUNDA_OPERATE_ZEEBEOPENSEARCH_PREFIX", indexPrefix);
+        put("ZEEBE_BROKER_GATEWAY_ENABLE", "true");
+        put("CAMUNDA_DATABASE_URL", "http://opensearch:9200");
+        // Reduce importer intervals to speed up tests
+        put("CAMUNDA_OPERATE_IMPORTER_IMPORTPOSITIONUPDATEINTERVAL", "200");
+        put("CAMUNDA_OPERATE_IMPORTER_READERBACKOFF", "200");
+        put("CAMUNDA_TASKLIST_IMPORTER_IMPORTPOSITIONUPDATEINTERVAL", "200");
+        put("CAMUNDA_TASKLIST_IMPORTER_READERBACKOFF", "200");
+      }
+    };
+  }
+
+  private void extractVolume() {
+    try {
+      volume.extract(zeebeDataPath);
+    } catch (final IOException e) {
+      throw new RuntimeException(e);
```

Comment:
:wrench: You can wrap using `UncheckedIOException`, a standard JVM exception specifically made for wrapping `IOException`.